src/libcore/scanAccounts.js

```diff
@@ -176,6 +179,15 @@ export const scanAccounts = ({
 
             let result;
 
+            if (derivationMode === "native_segwit") {
+              const { name, version } = await getAppAndVersion(transport);
```

Comment:
can you move this logic inside the if below?

Reply:
done